graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/CqlDirectives.java

```diff
@@ -583,6 +584,15 @@
           .validLocation(ARGUMENT_DEFINITION)
           .build();
 
+  private static final GraphQLDirective TIMESTAMP_DIRECTIVE =
+      newDirective()
+          .name(TIMESTAMP)
+          .description(
+              "Annotates a parameter to indicate that it will be used as a write timestamp for this row."
+                  + "This is only allowed for insert and update mutations. It must be a long value.")
```

Comment:
Needs an update to mention ISO strings. Also it needs to be more descriptive for people not familiar with CQL and our custom scalars:
```suggestion
                  + "This is only allowed for insert and update mutations. The parameter can be "
                  + "either a `BigInt` (that represents a number of microseconds since the epoch), "
                  + "or a `String` (that represents an ISO-8601 zoned date time, e.g. "
                  + "`2007-12-03T10:15:30+01:00`).")
```